pkg/webhook/validator.go

```diff
@@ -846,8 +912,8 @@ func getNamespace(ctx context.Context, namespace string) string {
 // All the matched policies were validated, or
 // no matching policies were found, but the PolicyControllerConfig has been
 // configured to allow images not matching any policies.
-func (v *Validator) validateContainer(ctx context.Context, c corev1.Container, namespace, field string, index int, kind, apiVersion string, labels map[string]string, ociRemoteOpts ...ociremote.Option) *apis.FieldError {
-	ref, err := name.ParseReference(c.Image)
+func (v *Validator) validateContainer(ctx context.Context, containerImage string, namespace, field string, index int, kind, apiVersion string, labels map[string]string, ociRemoteOpts ...ociremote.Option) *apis.FieldError {
```

Comment:
nit, wonder if we should now rename this (internal so should be fine) to:
validateContainerImage, or just validateImage?

Reply:
I agree, somehow i don't know why i missed to rename it. good catch